Pheno/scripts/QC_script.R

```diff
@@ -67,5 +62,7 @@ for (row in seq_len(nrow(data))) {
   data[row, ]$Final_M <- avg_M
 }
 
+nrow(data)
```

Comment:
i dont think this is needed?

Reply:
I wanted it for visual to compare how many individuals were removed in the process of QC

Reply:
i see. I dont think it's adding to the actual functionality of the file however. If you ever need to visualize something like this you can simply run `nrow(data)` in your RStudio console. But here it's simply bloat

Reply:
ok... ur right